README.md

```diff
@@ -1,7 +1,52 @@
 # FlowControllerKit
 
-## Feature 
+[![Build Status](https://travis-ci.org/varvet/FlowControllerKit.svg?branch=master)](https://travis-ci.org/varvet/FlowControllerKit)
 
-This framework contains methods that can help with implementing the FlowController/FlowCoordinator patern.
+A collection of helpers to make cleaner Flow Controller classes.
 
+If you have ever been bothered by the tight data and navigation coupling of UIViewControllers when using storyboard and segues the Flow Controller pattern could be something worth looking in to.
 
+Flow Controller (aka Flow Coordinators) is when you centralise the navigation and data injection to another class that handle just that. It makes sure that the right view controller is shown with the right data.
```

Comment:
centralize -> centralize

"handle just that" - we should clarify or just skip that part of the sentence.

I would use "It makes sure that the _correct_ view controller..." to prevent confusion with the directional usage of the word